packages/vscode-extension/package.json

```diff
@@ -98,6 +101,18 @@
     "configuration": {
       "title": "Shopify Liquid | Syntax Highlighting & Linter by Shopify",
       "properties": {
+        "shopifyLiquid.trace.server": {
+          "type": [
+            "string"
+          ],
+          "description": "Traces the communication between the client and the server.",
+          "default": "off",
+          "enum": [
+            "off",
+            "messages",
+            "verbose"
+          ]
+        },
```

Comment:
Added that stuff in there because I was tired of not having code completion in settings.json to enable the LSP logging